### PR TITLE
fix: 修复tooltip 点击空白无法消失的问题

### PR DESCRIPTION
**Pull request by stone-lyl**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [X] Solve the issue and close #0

🔧 Chore

- [X] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 📝 Description
- fix: 修复不按类型引入，导致 `react:playground` 启动报错
- [fix: 修复tooltip 点击空白无法消失的问题](https://github.com/antvis/S2/commit/4a604fcd825cb3308eeef76ae6e6d002453e3c35)
### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |
|image |image|
|error-tooltip|right-tooltip|

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-core/src/interaction/event-controller.ts [hunk 1]
```diff
@@ -162,7 +162,13 @@ export class EventController {
     // 所以如果是 刷选过程中 引起的 click(mousedown + mouseup) 事件, 则不需要重置
     const { interaction } = this.spreadsheet;
 
-    if (this.hasBrushSelectionIntercepts()) {
+    if (
+      interaction.hasIntercepts([
```
Comment (lijinke666) [inline packages/s2-core/src/interaction/event-controller.ts:166]: 为啥不直接修改 hasBrushSelectionIntercepts 这个方法？

Comment (stone-lyl) [reply packages/s2-core/src/interaction/event-controller.ts:166]: 因为其他地方要用啊

**Review by lijinke666 — APPROVED** [review]

**Pull request merged**